File: eTeatar/eTeatar.Model/Karta.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTeatar.Model
{
    public class Karta
    {
        public int KartaId { get; set; }

        public decimal Cijena { get; set; }

        public int SjedisteId { get; set; }

        public int TerminId { get; set; }

        public int? RezervacijaId { get; set; }

        public int KorisnikId { get; set; }
    }
}

File: eTeatar/UI/eteatar_mobile/lib/models/karta.dart
import 'package:json_annotation/json_annotation.dart';

part 'karta.g.dart';

@JsonSerializable()
class Karta {
  int? kartaId;
  double? cijena;
  int? sjedisteId;
  int? terminId;
  int? rezervacijaId;
  int? korisnikId;
  Karta(
    this.kartaId,
    this.cijena,
    this.sjedisteId,
    this.terminId,
    this.korisnikId
  );

  factory Karta.fromJson(Map<String, dynamic> json) => _$KartaFromJson(json);

  /// Connect the generated [_$GlumacToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$KartaToJson(this);
}